### ISSUE TITLE
BEIT2 ADE20 fine-tuning results do not match the accuracy reported in the github

### ISSUE THREAD

**Issue by jsrdcht** [microsoft/unilm#1110]
**Describe the bug**
Model I am using (UniLM, MiniLM, LayoutLM ...): BEIT 2

The problem arises when using:
* [-] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

I downloaded beitv2_base_patch16_224_pt1k on Github, downloaded the code for semantic segmentation, and ensured consistent environment through Docker. However, I only obtained 52.63 in fine-tuning. Even when I tried multiple experiments with my own environment, the highest score I got was only 52.8. However, Github shows that there should be 53.1 with direct fine-tuning.

**Comment by caddyless** [comment]
I met the same issue exactly. I finetuned the same checkpoint strictly following the official instructions, but only got 52.26 mIoU finally, which is far behind the reported one 53.1. I have checked the provided log file and found that all configurations are consistent except the pytorch version (provided 1.6.0, reproduced 1.7.1). I would appreciate it if anyone could give me some insights.
